schemas/dart_model.schema.json

```diff
@@ -117,6 +117,31 @@
         "returnType": {
           "$comment": "The return type of this member, if it has one.",
           "$ref": "#/$defs/StaticTypeDesc"
+        },
+        "requiredPositionalParameters": {
```

Comment:
I wouldn't separate the required from the optional positional parameters - I would look at the existing introspection APIs for declarations when adding them here, for example https://github.com/dart-lang/sdk/blob/main/pkg/_macros/lib/src/api/introspection.dart#L269. These have been fairly well fleshed out at this point and should be directly copied imo.